modules/vectorfieldvisualization/include/modules/vectorfieldvisualization/processors/2d/seedpointgenerator2d.h

```diff
@@ -51,6 +51,7 @@ namespace inviwo {
 class IVW_MODULE_VECTORFIELDVISUALIZATION_API SeedPointGenerator2D : public Processor {
 public:
     enum class Generator { Random, HaltonSequence };
+    enum class SamplingDomain { FullDomain, Line, Rectangle, Disk };
```

Comment:
Would SamplingGeometry be more representative?

FullDomain is rather UnitSquare?

Reply:
`FullDomain` was chosen in connection with potential extensions toward other coordinate spaces, not just [0,1]. Especially since using [0,1] will often distort the actual shape.